SemiTruck.h

```diff
@@ -1,2 +1,15 @@
 #pragma once
 
+#include "Vehicle.h"
+
+struct SemiTruck : public Vehicle
```

Comment:
10) correct any unnecessary keywords related to access specifiers used in the UDTs.
- [ ] revise